Gold-Dataset-EDA.ipynb

```diff
@@ -0,0 +1,1584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plot_timeseries(df,'Price', 260) # for one year
</pre><p>If you want to be more accurate you can rely on calender to get the number of wrok days per specific year, it is not a big deal since the difference migth be one or two days</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/shamelj/Precious-Metals-Forecaster/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>